content/user/first_steps/how_do_i_install.md

```diff
@@ -159,21 +159,6 @@ already have a well established website are just adding on ecommerce using
 Zen Cart.  In that case, you'd want to put your store below your existing site. 
 
 ---
-## C. Create the configure.php files
```

Comment:
Why was this removed?

Reply:
No longer needed.  It happens automatically.